src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -731,4 +731,84 @@ public void testGetDuplicatedItems(final Collection<?> collection, final Set<?>
+                    new Object[] { stringData, "f",   Arrays.asList( stringData.split("f"  ) ) },
+                    new Object[] { stringData, "g",   Arrays.asList( stringData.split("g"  ) ) },
+                    new Object[] { stringData, "h",   Arrays.asList( stringData.split("h"  ) ) },
+                    new Object[] { stringData, "i",   Arrays.asList( stringData.split("i"  ) ) },
+                    new Object[] { stringData, "j",   Arrays.asList( stringData.split("j"  ) ) },
+                    new Object[] { stringData, "k",   Arrays.asList( stringData.split("k"  ) ) },
+                    new Object[] { stringData, "l",   Arrays.asList( stringData.split("l"  ) ) },
+                    new Object[] { stringData, "m",   Arrays.asList( stringData.split("m"  ) ) },
+                    new Object[] { stringData, "n",   Arrays.asList( stringData.split("n"  ) ) },
+                    new Object[] { stringData, "o",   Arrays.asList( stringData.split("o"  ) ) },
+                    new Object[] { stringData, "p",   Arrays.asList( stringData.split("p"  ) ) },
+                    new Object[] { stringData, "q",   Arrays.asList( stringData.split("q"  ) ) },
+                    new Object[] { stringData, "r",   Arrays.asList( stringData.split("r"  ) ) },
+                    new Object[] { stringData, "s",   Arrays.asList( stringData.split("s"  ) ) },
+                    new Object[] { stringData, "t",   Arrays.asList( stringData.split("t"  ) ) },
+                    new Object[] { stringData, "u",   Arrays.asList( stringData.split("u"  ) ) },
+                    new Object[] { stringData, "v",   Arrays.asList( stringData.split("v"  ) ) },
+                    new Object[] { stringData, "w",   Arrays.asList( stringData.split("w"  ) ) },
+                    new Object[] { stringData, "x",   Arrays.asList( stringData.split("x"  ) ) },
+                    new Object[] { stringData, "y",   Arrays.asList( stringData.split("y"  ) ) },
+                    new Object[] { stringData, "z",   Arrays.asList( stringData.split("z"  ) ) },
+                    new Object[] { stringData, " ",   Arrays.asList( stringData.split(" "  ) ) },
+                    new Object[] { stringData, "T",   Arrays.asList( stringData.split("T"  ) ) },
+                    new Object[] { stringData, "9",   Arrays.asList( stringData.split("9") ) }
+                )
+        );
+
+        // Create test cases for words:
+        for ( final String delim : wordsToSplitOn ) {
+            testCases.add( new Object[] { stringData, delim, Arrays.asList( stringData.split(delim) ) } );
+        }
+
+        // Create test cases for repeated substrings:
+        for ( final String delim : repeatedSubstringsToSplitOn ) {
+            testCases.add( new Object[] { stringData, delim, Arrays.asList( stringData.split(delim) ) } );
+        }
+
+        return testCases.toArray( new Object[][] {} );
```

Comment:
You can avoid copying here by making the dataprovider return `Iterator` and return `testCases.iterator` instead.

Reply:
Interesting. 

Fixed!